tests/test_docs.py

```diff
@@ -0,0 +1,32 @@
+import os.path
+import re
+import textwrap
+
+import pytest
+
+from tox.config import parseconfig
+
+INI_BLOCK_RE = re.compile(
+    r"(?P<before>"
+    r"^(?P<indent> *)\.\. (code-block|sourcecode):: ini\n"
+    r"((?P=indent) +:.*\n)*"
+    r"\n*"
+    r")"
+    r"(?P<code>(^((?P=indent) +.*)?\n)+)",
+    re.MULTILINE,
+)
+
+
+rst_files = []
+for root, _, filenames in os.walk("doc"):
```

Comment:
does pytest guarantee the current working directory is the root; or should we do some kind of relative from this file logic here?

Reply:
it doesn't if some other test runs `chdir` then this points at the wrong file locations -- I can `abspath` + join with `__file__` to make this better

Reply:
Yeah I think that would make this more robust; otherwise looks good :+1:

Reply:
Also in the CI I see:

```
SKIP [1] tests/test_docs.py:27: got empty parameter set ['filename'], function test_all_rst_ini_blocks_parse at /home/travis/build/tox-dev/tox/tests/test_docs.py:26
```

Reply:
heh, probably due to the `changedir`